workflows-fiji.html

```diff
@@ -0,0 +1,507 @@
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Edit the following:</p>
+
+<p><span class="bold">Line 131:</span><br/>
+From:&nbsp;&nbsp;&nbsp;<span class="code-example">HOST = &#34;omero-latest-analysis.docker.openmicroscopy.org&#34;</span><br/>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">HOST = &#34;your.omero.server.url&#34;</span></p>
+
+<p><span class="bold">Line 133:</span><br/>
+From:&nbsp;&nbsp;&nbsp;<span class="code-example">datasetId = &#34;1&#34;</span><br/>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">datasetId = &#34;x&#34;</span></p>
+
+<p>Where <span class="code-example">&#34;x&#34;</span> is an OMERO Dataset ID of your choice.</p>
+
+<p><span class="bold">Line 139:</span><br/>
+From:&nbsp;&nbsp;&nbsp;<span class="code-example">CREDENTIALS = &#34;/Users/bramalingam/Desktop/FijiDemonstration/credentials.txt&#34;</span><br/>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">CREDENTIALS = &#34;PATH_TO/credentials.txt&#34;</span></p>
+
+<p><span class="bold">Line 142:</span><br/>
+From:&nbsp;&nbsp;&nbsp;<span class="code-example">macroFilePath = &#34;/Users/bramalingam/Desktop/FijiDemonstration/bg_subtract.ijm&#34;</span><br/>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">macroFilePath = &#34;PATH_TO/bg_subtract.ijm&#34;</span></p>
+
+<p><span class="bold">Line 145:</span><br/>
+From:&nbsp;&nbsp;&nbsp;<span class="code-example">paths = &#34;Users/bramalingam/Desktop/FijiDemonstration&#34;</span><br/>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">paths = &#34;PATH_TO&#34;</span></p>
+
+<p>Where <span class="code-example">&#34;PATH_TO&#34;</span> is the file path on your system to the files you downloaded previously.</p>
+
+</li>
+
+<li><p class="first">Use <span class="bold">File &#62; Save</span> to save the script file.</p>
+
+</li>
+
+<li><p class="first">Use <span class="bold">File &#62; Open...</span> to select and open the <span class="code-example">credentialsTemplate.txt</span> file.</p>
+
+</li>
+
+<li><p class="first">Enter the following using the OMERO login credentials you have been given:</p>
+
+<p><span class="bold">Line 1:</span></span><br/>
+From:&nbsp;&nbsp;&nbsp;<span class="code-example">USERNAME = &#60;OMERO USERNAME&#62;</span><br/>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;USERNAME = &#60;user-x&#62;</span></p>
+
+<p><span class="bold">Line 2:</span><br/>
+From:&nbsp;&nbsp;&nbsp;<span class="code-example">PASSWORD = &#60;OMERO PASSWORD&#62;</span><br/>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PASSWORD = &#60;ome&#62;</span></p>
```

Comment:
ome as the password? Shouldn't it be user-x's password or some such to be clearer?